src/components/apiErrorAlert/apiErrorCode.ts

```diff
@@ -51,13 +52,15 @@ export enum ApiErrorCode {
     HAR_ALLEREDE_EN_ÅPEN_SØKNADSBEHANDLING = 'har_allerede_en_åpen_søknadsbehandling',
     HAR_IKKE_EKTEFELLE = 'har_ikke_ektefelle',
     HELE_BEHANDLINGSPERIODEN_MÅ_HA_VURDERING = 'hele_behandlingsperioden_må_ha_vurderinger',
+    HENTET_JOURNALPOST_IKKE_KNYTTET_TIL_SAK = 'hentet_journalpost_ikke_knyttet_til_sak',
```

Comment:
```suggestion
    HENTET_JOURNALPOST_IKKE_KNYTTET_TIL_SAK = 'journalpost_ikke_knyttet_til_sak',
```
Må endre bakover og.